### PR TITLE
Added redirects for Attribution Reporting

### PR DESCRIPTION
**Pull request by maudnals**
Attribution Reporting API content has moved to dcc some time ago. 
The two web.dev articles have long been obsolete; the API they document don't work for Chrome 87+.

Changes proposed in this pull request:
- Redirect the two legacy web.dev articles to the dcc landing page for this API
- Unlist the two legacy web.dev articles from the web.dev/secure overview

### CODE REVIEW

**Comment by maudnals** [comment]
This is ready to merge.

**Comment by mihajlija** [comment]
Hm, doesn't seem to be working. Both URLs still load the web.dev articles. Could be a staging thing, I'll ask around.

https://deploy-preview-6834--web-dev-staging.netlify.app/using-conversion-measurement/
https://deploy-preview-6834--web-dev-staging.netlify.app/using-conversion-measurement/

File: redirects.yaml [hunk 1]
```diff
@@ -300,3 +300,9 @@ redirects:
   # Design system
   - source: /design-system/component/action
     destination: /design-system/component/fab
+
+  # Redirect for Attribution Reporting API content (ex-conversion measurement) moved to dcc
+  - source: /conversion-measurement/
```
Comment (devnook) [inline redirects.yaml:305]: Let's keep it together with redirects in line 288 (Redirects to https://developer.chrome.com/.)

File: src/site/_data/paths/secure.json [hunk 2]
```diff
@@ -48,8 +48,6 @@
         "samesite-cookie-recipes",
         "referrer-best-practices",
         "trust-tokens",
-        "conversion-measurement",
```
Comment (devnook) [inline src/site/_data/paths/secure.json:51]: Does this mean that the correspondinf articles should be removed too? e.g. https://github.com/GoogleChrome/web.dev/blob/main/src/site/content/en/blog/conversion-measurement/index.md

Comment (maudnals) [reply src/site/_data/paths/secure.json:51]: Yes, these can be removed

**Comment by devnook** [comment]
I think you need to remove the posts .md file for the redirect to work on netlify, see here https://github.com/GoogleChrome/web.dev/blob/main/redirects.yaml#L11

**Comment by maudnals** [comment]
> I think you need to remove the posts .md file for the redirect to work on netlify, see here https://github.com/GoogleChrome/web.dev/blob/main/redirects.yaml#L11

Sounds good, done

**Review by mihajlija — APPROVED** [review]

File: redirects.yaml (hunk 1 above)
Comment (mihajlija) [reply redirects.yaml:305]: @maudnals PR is approved, so once you move the code a bit further up as Ewa suggested you can merge.

Comment (maudnals) [reply redirects.yaml]: Moved, I'll merge once the build succeeds

**Pull request merged**